resource.html

```diff
@@ -1325,13 +1291,22 @@ <h2 class="hover-underline-animation">Regional Language Book Section</h2>
             <div class="lower">
                 <div class="social-container">
                     <div class="social-icons">
+
+                            <a class="Twitter" href="https://twitter.com/mesourav44" target="_blank"><i class="fab fa-twitter"></i></a>
+                            <a class="Instagram" href="https://www.instagram.com/mesouravofficial/" target="_blank"><i class="fab fa-instagram"></i></a>
+                            <a class="linkedin" href="https://www.linkedin.com/in/sauravmukherjee44/" target="_blank"><i class="fab fa-linkedin"></i></a>
+                            <a class="Github" href="https://github.com/SauravMukherjee44/" target="_blank"><i class="fab fa-github-square"></i></a>
+                            <a class="Blog" href="#" target="_blank"><i class="fa fa-rss-square"></i></a>
+                    </div>          
+
```

Comment:
Why has this been added ?

Reply:
I apologize for the misunderstanding. This code snippet was not added by me, it was already their. If you find the code snippet unnecessary for the project, please let me know,  I'll remove it.

Reply:
I thinks its been added by you(as it shows over here) maybe when you were merging main you handled it like this 
Bcoz the code of this is already below...

Reply:
So needs to be removed

Reply:
The code snippet has been successfully removed.
I kindly request you to review the updated Pull Request and consider merging it into the main branch if everything looks good.